Modules/Sources/Yosemite/PointOfSale/OrderList/POSOrderRefundCondensed.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 import struct NetworkingCore.OrderRefundCondensed
 import class WooFoundationCore.CurrencyFormatter
 
-public struct POSOrderRefund: Equatable, Hashable {
+public struct POSOrderRefundCondensed: Equatable, Hashable {
```

Comment:
Why have we renamed `POSOrderRefund` to `POSOrderRefundCondensed`? This seems to leak the aspect/shape/UI of a refund into the model, and would imply that there is a non-condensed version as well. 

I know we do that in the non-POS side of the app by using `OrderRefundCondensed` in the `Order` model, but perhaps this is not translatable directly to POS and we could use the `POSOrderRefund` model until there is a need to disambiguate it further

I haven't looked much into it, but at first glance I think it could make sense to just make the necessary transformations in the mapper or a close layer from WooCommerce `[Refund]` to POS `[POSOrderRefund]` and then further condensations/transformations can be done at the UI layer.